### PR TITLE
If "project type" didn't appear, install extension

### PR DESCRIPTION
**Pull request by Stiles-X**

### CODE REVIEW

**Comment by PRMerger13** [comment]
@Stiles-X : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Stiles-X** [comment]
any progress?

File: docs/reporting-services/lesson-1-creating-a-report-server-project-reporting-services.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ When you create reports with Report Designer, it creates a report server project
...
+    > For VS, if you don't see Reporting Services in the left column, add the Report Designer by installing the SSDT workload. From the **Tools** menu, select **Get Tools and Features...** and select the **SQL Server Data Tools** from the workloads displayed. If you don't see the Report Services objects in the center column, add the Reporting Services extensions. From the **Tools** menu, select **Extensions and Updates** > **Online**. In the center column, select **Microsoft Reporting Services Projects** > **Download** from the displayed extensions. For SSDT, See [Download SQL Server Data Tools (SSDT)](../ssdt/download-sql-server-data-tools-ssdt.md). In Visual Studio 2019, and previous steps did not work, try installing [Microsoft Reporting Service Project extension](https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftReportProjectsforVisualStudio).
```
Comment (maggiesMSFT) [inline docs/reporting-services/lesson-1-creating-a-report-server-project-reporting-services.md:34]:
```suggestion
    > For VS, if you don't see Reporting Services in the left column, add the Report Designer by installing the SSDT workload. From the **Tools** menu, select **Get Tools and Features...** and select the **SQL Server Data Tools** from the workloads displayed. If you don't see the Report Services objects in the center column, add the Reporting Services extensions. From the **Tools** menu, select **Extensions and Updates** > **Online**. In the center column, select **Microsoft Reporting Services Projects** > **Download** from the displayed extensions. For SSDT, See [Download SQL Server Data Tools (SSDT)](../ssdt/download-sql-server-data-tools-ssdt.md). In Visual Studio 2019, and previous steps did not work, try installing [Microsoft Reporting Service Project extension](https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftReportProjectsforVisualStudio).
```suggestion
    > For VS, if you don't see Reporting Services in the left column, add the Report Designer by installing the SSDT workload. From the **Tools** menu, select **Get Tools and Features...** and select the **SQL Server Data Tools** from the workloads displayed. If you don't see the Report Services objects in the center column, add the Reporting Services extensions. From the **Tools** menu, select **Extensions and Updates** > **Online**. In the center column, select **Microsoft Reporting Services Projects** > **Download** from the displayed extensions. For SSDT, See [Download SQL Server Data Tools (SSDT)](../ssdt/download-sql-server-data-tools-ssdt.md). In Visual Studio 2019, if previous steps didn't work, try installing [Microsoft Reporting Service Project extension](https://marketplace.visualstudio.com/items?itemName=ProBITools.MicrosoftReportProjectsforVisualStudio).

```
```

**Comment by maggiesMSFT** [comment]
Hey, @ktoliver , can you accept my changes? Then I can sign off. 
Thanks!

**Comment by maggiesMSFT** [comment]
Hi, @Stiles-X -- your change is in progress. Thanks!

**Comment by PRMerger17** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
> Hey, @ktoliver, can you accept my changes? Then I can sign off.
> Thanks!

@maggiesMSFT Done.

**Review by maggiesMSFT — APPROVED** [review]
Thanks, @Stiles-X and @ktoliver

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by ktoliver** [comment]
Thanks!

**Comment by maggiesMSFT** [comment]
Thanks!

**Pull request merged**